tests/test_git.py

```diff
@@ -67,3 +75,52 @@ def test_extract_commit_message_metadata():
     metadata = git.extract_commit_message_metadata(message)
 
     assert metadata == {"One": "Hello!", "Two": "1234"}
+
+
+class TestClone(unittest.TestCase):
+    def setUp(self):
+        # Preserve the original environment variables.
+        self.env = copy.copy(os.environ)
+        return super().setUp()
+
+    def tearDown(self):
+        os.environ = self.env
+        return super().tearDown()
+
+    def testClone(self):
+        local_directory = git.clone("https://github.com/googleapis/nodejs-vision.git")
```

Comment:
by default what directory path will this be cloned into, curious if it also ends up in `tmp`.

Reply:
It gets cloned into `~/.cache/synthtool/nodejs-vision`